01_stratified_classification/nbconverted/plot_univariate_fs_results.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Analysis of feature selection results
+# 
+# We tried a few different feature selection methods for mutation prediction:
+# 
+# * mean absolute deviation
+# * pan-cancer univariate f-statistic (on training data, separate samples into mutated/not mutated and do an f-test)
+# * median univariate f-statistic across cancer types (same as above for each individual cancer type, then take median)
+# * MAD of univariate f-statistic across cancer types (same as above, but look for least variable genes)
+# 
+# Here, we're just looking at stratified CV across all cancer types, as a sanity check - the idea is to make sure the univariate correlation-based feature selection methods perform reasonably well. This isn't necessarily where we'd expect them to help performance, but we still don't want them to _hurt_ it.
+
+# In[1]:
+
+
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import upsetplot as up
+from venn import generate_petal_labels
+
+import pancancer_evaluation.config as cfg
+import pancancer_evaluation.utilities.analysis_utilities as au
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+
+# ### Load results and look at overall performance
+
+# In[2]:
+
+
+results_dir = os.path.join('results', 'univariate_fs', 'pancancer')
```

Comment:
In this section maybe a quick "looking at results that got generated by this script" pointer